lib/usd/hdMaya/adapters/dagAdapter.h

```diff
@@ -57,7 +57,7 @@ class HdMayaDagAdapter : public HdMayaAdapter
     HDMAYA_API
     virtual void RemovePrim() override;
     HDMAYA_API
-    const GfMatrix4d& GetTransform();
+    GfMatrix4d GetTransform();
```

Comment:
I'm guessing this was changed from `const GfMatrix4d &` to `GfMatrix4d` just to make it more consistent with `HdSceneDelegate::GetTransform`?

Reply:
Pretty much.  The reference was always copied anyway.